app/components/summary_list_component.rb

```diff
@@ -31,7 +31,7 @@ def action(row)
       end
       links.join('<br>').html_safe
     elsif any_row_has_action_span?
-      tag.dd(class: 'govuk-summary-list__actions')
+      nil
```

Comment:
Could we remove the whole `any_role_has_action_span?` clause? Method will return `nil` by default

Reply:
That’s what I tied originally, but the tests failed. I think it’s because we need to explicitly check for `any?`, which is what `any_role_has_action_span?` method does.

Reply:
That test fails with the current implementation too! Isn't it up to the govuk-component to look at all the rows and decide how to render? ie providing this logic isn't our job any more

Reply:
Are you saying I should delete the test? That’s my favourite kind of feedback (and this is the second time that’s happened in as many weeks 😂)

Reply:
🔪 🔪 🔪 !

Reply:
I love the smell of deleted code in the morning (or afternoon).